web/src/components/Firebase/Firebase.js

```diff
@@ -108,7 +116,13 @@ const FirebaseUserTools = () => {
 
 export default (props) => {
   return (
-    <AuthProvider client={firebaseClient} type="firebase" {...props}>
+    // skipFetchCurrentUser since backend does not have firebase admin app setup
+    <AuthProvider
+      client={firebaseClient}
+      type="firebase"
+      skipFetchCurrentUser
```

Comment:
building grahpql endpoint is failing, which I believe has to do with transition from v0.36 to v0.37 redwood

```
api | Importing Server Functions...
api | (node:81469) UnhandledPromiseRejectionWarning: TypeError: Cannot convert undefined or null to object
api |     at Function.values (<anonymous>)
api |     at makeMergedSchema (/Users/omar/src/playground-auth/node_modules/@redwoodjs/graphql-server/dist/makeMergedSchema/makeMergedSchema.js:131:29)
api |     at Object.<anonymous> (/Users/omar/src/playground-auth/api/src/functions/graphql.js:20:11)
api |     at Module._compile (internal/modules/cjs/loader.js:1085:14)
api |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
api |     at Module.load (internal/modules/cjs/loader.js:950:32)
api |     at Function.Module._load (internal/modules/cjs/loader.js:784:16)
api |     at Module.require (internal/modules/cjs/loader.js:974:19)
api |     at require (internal/modules/cjs/helpers.js:92:18)
api |     at /Users/omar/src/playground-auth/node_modules/@redwoodjs/api-server/dist/middleware/withFunctions.js:57:11
api | (Use `node --trace-warnings ...` to show where the warning was created)
api | (node:81469) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
api | (node:81469) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```